src/react/components/common/assetPreview/videoAsset.test.tsx

```diff
@@ -55,6 +55,18 @@ describe("Video Asset Component", () => {
         expect(wrapper.state().loaded).toBe(false);
     });
 
+    it("does not subscribe to state change callback when autoPlay is false", () => {
```

Comment:
shouldn't we another test for when autoPlay is true, that "subscribe" is called?

Reply:
Almost every other test would fail were this not the case. The change (and this test) were to handle the explicit case, adding another test to handle the default case (which is covered numerous times in the other tests) seems to be unnecessary.